cfgov/jinja2/v1/ask-cfpb/landing-page.html

```diff
@@ -48,25 +57,49 @@ <h2 class="u-show-on-mobile u-mt30">Browse questions by category</h2>
                                         <article class="m-card content-l_body__equal-height">
 
                                             <h2>
+                                                {% if page.language == 'es' %}
+                                                <a href="{{ slugurl( 'categoria-' + category.slug_es ) }}">
+                                                {% else %}
                                                 <a href="{{ slugurl( 'category-' + category.slug ) }}">
+                                                {% endif %}
                                                 {{ category_icon.render( category.name ) }}
+                                                {% if page.language == 'es' %}
+                                                {{ category.name_es }}
+                                                {% else %}
                                                 {{ category.name }}
+                                                {% endif %}
                                                 </a>
                                             </h2>
 
                                             <ul class="m-list question_list m-list__flush-left u-mt10 u-mb30">
-                                                {% for a in category.featured_answers('en') %}
-                                                <li class="m-list_item">
-                                                {{ partially_styled_link.render( {'plain_text': a.question, 'underlined_text': 'Read answer', 'url': a.url} ) }}
-                                                </li>
-                                                {% endfor %}
+                                                {% if page.language == 'es'%}
+                                                    {% for a in category.featured_answers('es') %}
```

Comment:
MINOR: `a` variable would be clearer as `answer`

Reply:
Updated in f06ddeeb84ff023e7b717dc3507e5ec1392e72c0